scripts/shoot_peng.py

```diff
@@ -200,36 +200,37 @@ def run_peng(args, output_directory, run_scoring):
         # run FDR
         fdr_command_line = build_fdr_command(args, args.fasta_file, peng_output_file, output_directory)
         subprocess.run(fdr_command_line, check=True, stdout=stdout)
-        r_output_file = os.path.join(output_directory, prefix + ".bmscore")
+
+        r_output_file = os.path.join(output_directory, prefix + ".aucscore")
```

Comment:
is it necessary to rename that file? It might be a bit of work to adapt this on the server.

Reply:
okay, I can change it back.

Reply:
but the content is changed. Instead of "ausfc auprc roc5 occur", it is changed to "data_aurrc data_occur motif_aurrc motif_occur"